modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -297,6 +321,11 @@ export default {
       }
     },
     menuHandler(action) {
+      const operation = this.customOperations.find(op => op.action === action);
+      if (operation) {
+        this._customAction(this.context, operation);
```

Comment:
We do not use the "_" to mean "private method" in Apostrophe, just use `customAction`

Reply:
This would end up badly when `{ action: 'customAction' }` is provided. But I can solve it/

Reply:
Actually it won't. Fixed.